x/auth/vesting/types/msgs.go

```diff
@@ -11,11 +11,12 @@ var (
 )
 
 // NewMsgCreateVestingAccount returns a reference to a new MsgCreateVestingAccount.
-func NewMsgCreateVestingAccount(fromAddr, toAddr sdk.AccAddress, amount sdk.Coins, endTime int64, delayed bool) *MsgCreateVestingAccount {
+func NewMsgCreateVestingAccount(fromAddr, toAddr sdk.AccAddress, amount sdk.Coins, startTime, endTime int64, delayed bool) *MsgCreateVestingAccount {
```

Comment:
can we get a changelog for this api break?

Reply:
I could also create a new function if we want to avoid the API change

Reply:
want to create the struct where its used instead of breaking this api? that way we dont break the api and dont expand the api surface